### ISSUE TITLE
Docker for Laravel dev - Setup video

### ISSUE THREAD

**Issue by bdjunayed** [laradock/laradock#774]
Docker for Laravel dev - easy
https://youtu.be/9HW1IOc1VMY

Hi, I made this, maybe it can help someone with the basic steps to install.

**Comment by winfried-van-loon** [comment]
Would be quite awesome to have a Laracasts-like video added to the Laradock documentation showcasing how Laradock works. 

I think the challenging part, to an official video, to keep it short (1-3 min) and interesting while being able to show what Laradock can do and why it's so cool, which basically comes down to:

* It should be an introduction video, not replace the docs
* Shouldn't be too specific, as the video shouldn't get outdated too quickly
* Showcase different environments (Caddy, NGINX, Apache2)
* To use in production and development
* Show the workspace functionality
* Invite viewers to send in issues/pull requests

**Comment by winfried-van-loon** [comment]
To have all information in a single issue: There are video's showcasing Laradock, made by Mahmoud Zalt as well. A link to the latest version: https://www.youtube.com/watch?v=TQii1jDa96Y

**Comment by Mahmoudz** [comment]
@bdjunayed great video, I'm sure it will help many people. Thanks.

@winfried-van-loon I used to make a new video after every major release. Last one was for v4. I hope I can record new one for v5 anytime soon. 

I like your idea of the videos. And I know people prefer watching videos instead of reading Docs. But I don't think we can make enough videos for Laradock as it's a small tool! But anything should be helpful, and would like to watch it :D Looking forward..
 
Note: recording screencasts videos is really time consuming. I built an educational platform called [Moubarmij](https://www.moubarmij.com/) _"means Programmer in Arabic"_ pretty similar to Laracasts in concept but not focused on Laravel only. Anyway I stopped making videos there because a 5 min video takes over 2 hour of preparation. writing a good into, deciding on the demo, repeating the same video 10 times hoping to make it better and more clear :P, editing the video at the end, upload to youtube, share it on social media......
